script/Ciel/Day 8/08_01_CIEL08_1C.txt

```diff
@@ -32,7 +32,7 @@ The second I call out to her, Senpai abruptly stops talking.
 -- Page 67, Offset 18427.
 -- 「よっ、おはよーサン。時間ギリギリで登校とか、おまえさんもここ一番でついてないね。今朝はシエル先輩が遊びにきてくれたってのに」
 // Sentence merged because ~のに is just so awkward to encode standalone
-"Yo, 'mornin!
+"Yo, 'mornin'!
```

Comment:
probably only need one of the single quotes here, otherwise it looks sarcastic

Reply:
I honestly didnt see the first one